### DISCUSSION TITLE
Python Based Share Service

### DISCUSSION THREAD

**Discussion by jbhoorasingh** [photobooth-app/photobooth-app#310] [Show and tell]
Hello PhotoBooth-App Community,

I am excited to introduce the new PhotoBooth-App ShareService, a Python implementation designed to enhance your photo-sharing experience. This service can be used instead of the original PHP-based version, offering a more robust and efficient solution with Flask and Boto3 for seamless integration with AWS S3.

Key Features
Easy Photo Downloads and Share - Download photos effortlessly through our intuitive web interface, share your photo with a variety of methods all from the webpage

Secure Storage with S3 (AWS/DigitalOcean) - Store your photos securely in an AWS S3 bucket using Boto3 for reliable and scalable storage, add lamda functions to each put operations

Lightweight and Efficient - The new implementation is lightweight, making it easy to deploy and manage, whether locally or in the cloud.

[photobooth-app-shareservice](https://github.com/jbhoorasingh/photobooth-app-shareservice )
[![Watch the video](http://i3.ytimg.com/vi/agiDfWkOLFE/hqdefault.jpg)](https://youtu.be/agiDfWkOLFE)

PS - I still have a few items to button up and tidy up the code. I only spend a few hours on this over the past two nights.

**Comment by mgineer85** [comment]
That’s cool 😎
